app/services/orderService.js

```diff
@@ -26,17 +27,14 @@ class OrderService {
         const ticker = await exchange.fetchTicker(strategy.symbol);
         const price = ticker.ask;
 
-        const markets = await exchange.loadMarkets();
-        const market = markets[strategy.symbol];
-        const precision = market.market.precision.amount;
-        //TODO: make sure to reach the minimum amount and prize
+        //TODO: make sure to reach the minimum amount and prize by const markets = await exchange.loadMarkets()[strategy.symbol];
 
         let amount = 0;
         // Fetch the current price, calculate purchase amount this time by type
         if (strategy.type === '1') {
```

Comment:
Better using a constant value or enum to define magic numbers or string without any meaning for the first look.

This will make the code more meaningful and readable.